tests/fb_directaccess/fb_directaccess_test.go

```diff
@@ -0,0 +1,128 @@
+package fbdirectaccess
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/scheduler/k8s"
+	. "github.com/portworx/torpedo/tests"
+	"os"
+	"testing"
+)
+
+const (
+	// TODO: having the secret key checked in the code probably bad, wonder whats the best way to keep this info?"
```

Comment:
should be ok for now as long as the account and user are dedicated for test only.

Reply:
Torpedo should have things in place for reading k8s secrets, could we put them in there? These really really really should not be here